guides/admin/timescale-migration.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Migrating data from TimescaleDB
+description: Learn how to migrate data from the TimescaleDB to a PostgreSQL
```

Comment:
Referring to the built-in database as `TimescaleDB` is a bit confusing because we no longer use Timescale, and it's just a regular database. Maybe we just refer to it as the built-in Postgres or bundled Postgres.

Reply:
makes sense - thing is, the default db is still referred to as `timescale-0` in the cluster. so it could get confusing for those not familiar with timescale.